src/dune

```diff
@@ -1,9 +1,17 @@
 (env
  (dev
+  ;; This profil is used (default) to dev without optimisation and blocking warnings.
   (flags
-   (:standard -bin-annot -w -22))
-  (ocamlopt_flags -g -O3 -unbox-closures))
+   (:standard -bin-annot -w -22 -warn-error -A))
+  (ocamlopt_flags -g))
+ (warnings
+  ;; This profile can be used to simply enable all warnings.
+  (flags
+   (:standard -bin-annot -w +A -warn-error -A))
+  (ocamlopt_flags -g))
  (release
+  ;; The release profile has optimisation enable.
   (flags
-   (:standard -bin-annot -w -22))
-  (ocamlopt_flags -O3 -unbox-closures)))
+   (:standard -bin-annot -w -A+K+X+Y+Z-22 -warn-error +X+K+Y+Z))
```

Comment:
This is wrong, as it will very likely break the build on future releases of the compiler, especially considering the face that alt-ergo is released as sources rather than pre-compiled binaries. I'd very much prefer that for the release profile, you keep the dune defaults, which are good on this case.

Reply:
Fixed